src/EditorFeatures/Core/Extensibility/Completion/ICompletionPresenterSession.cs

```diff
@@ -14,7 +14,7 @@ internal interface ICompletionPresenterSession : IIntelliSensePresenterSession
             ITrackingSpan triggerSpan, IList<CompletionItem> items, CompletionItem selectedItem,
             CompletionItem suggestionModeItem, bool suggestionMode, bool isSoftSelected,
             ImmutableArray<CompletionItemFilter> completionItemFilters,
-            string filterText);
+            string filterText, Document document);
```

Comment:
FYI to @KirillOsenkov that this will impact Xamarin next time they take an update. Impact will be trivial, but just giving him a heads up.